airflow/config_templates/default_airflow.cfg

```diff
@@ -434,6 +434,10 @@ force_log_out_after = 0
 # The UI cookie lifetime in days
 session_lifetime_days = 30
 
+# Have the webserver save the dagbag to the DB every refresh. Set to true if you're
+# running without a scheduler and want DAGS to show up in the webserver
+locally_sync_dagbag = False
```

Comment:
The "locally" part tripped me up for a second - maybe just "sync_dagbag_to_db"?

Reply:
names are hard.